### DISCUSSION TITLE
Request for Assistance  regarding GitHub Student Developer Pack getting Rejected for Online Course

### DISCUSSION THREAD

**Discussion by vppai1993** [community/community#141816] [GitHub Education]
### Hi everyone,

I recently applied for the GitHub Student Developer Pack using my enrollment in the online distance learning program. I’ve attached my enrollment receipt as proof of registration.

However, my application was rejected with the following reasons (as shown in the attached screenshot):

![Screenshot_2024-10-17-23-41-38-81_e307a3f9df9f380ebaf106e1dc980bb6](https://github.com/user-attachments/assets/b55e58bd-f606-4214-bbc5-a9097f14ae3b)

Has anyone else experienced this issue for online courses, and what steps did you take to resolve it? Any assistance or guidance on how I can resolve this would be really helpful. I’ve already ensured that I am not using a VPN and have configured my system to allow location access.
Also i have created a GitHub education support case #3053572.

Thank you,

**Comment by Unmesh100** [comment]
screenshot.png
UG
fix it

Based on GitHub's rejection message, here are the specific steps to resolve your GitHub Student Developer Pack application for an online/distance learning program:

Email Verification:
Use your official academic email address from your school
Verify this email address on your GitHub account
Make sure to select your school when applying
Documentation:
Submit documentation that explicitly states this is a "virtual learning" or "online distance learning" program
The document should include:
Your name
School name
Current enrollment status
Explicit mention that it's an online/virtual program
Current academic term/year
Submit Support Ticket:
Since you've already created support case #3053572, add the following information:
Select the option "My selected school has incorrect or incomplete information e.g. domains or campus location"
Explain that you're enrolled in a legitimate online distance learning program
Attach the updated documentation showing virtual learning status
Location Settings:
You've already done this correctly by:
Disabling VPN
Allowing location access
Additional Tips:
Request an official letter from your school's administration specifically mentioning:
The program is officially delivered online/virtually
Your current enrollment status
The academic term dates
Make sure all documentation is current (less than 3 months old)
Once you've gathered all these materials, update your existing support ticket rather than creating a new application. This will help maintain continuity in your case.
